OperatingSystems.md

```diff
@@ -373,6 +379,12 @@ Identification|`__MSDOS__`
 Identification|`_MSDOS`
 Identification|`__DOS__`
 
+## [MSX](http://en.wikipedia.org/wiki/MSX) ##
+
+Type|Macro|Format|Description
+---|---|---|---
```

Comment:
Usually, additional macros added as separate line on the table.

Reply:
This is the style done for BSD, MacOS, UNICOS. I've switched these new ones to use multiple lines.